main_app/views.py

```diff
@@ -17,47 +19,50 @@
 from dotenv import load_dotenv
 
 load_dotenv()
-from .forms import UpdateUserForm, UpdateProfileForm
 
-from .models import Event, ViewingParty, Profile, Photo, User
 
 S3_BASE_URL = 'https://s3-us-west-1.amazonaws.com/'
 BUCKET = 'cosmos-app'
 NASA_API = str(os.getenv('NASA_API'))
 MAPBOX_API = str(os.getenv('MAPBOX_API'))
 # Create your views here.
 
+
 def home(request):
-    data = requests.get(f'https://api.nasa.gov/planetary/apod?api_key={NASA_API}').text
+    data = requests.get(
+        f'https://api.nasa.gov/planetary/apod?api_key={NASA_API}').text
```

Comment:
Some more weird line breaks happening here.

Reply:
finally figured out how to change this setting in my vscode autopep8 formatting on python 😅